dist/where-are-we.0.0.1.js

```diff
@@ -1 +0,0 @@
e]:{},r=Object.keys(n);"function"==typeof Object.getOwnPropertySymbols&&(r=r.concat(Object.getOwnPropertySymbols(n).filter(function(t){return Object.getOwnPropertyDescriptor(n,t).enumerable}))),r.forEach(function(e){P(t,e,n[e])})}return t}({},function(t){if(t&&t.loc&&"string"==typeof t.loc){var e=t.loc.split(",");if(2===e.length)return{lat:parseFloat(e[0]),lng:parseFloat(e[1])}}}(t),{commonName:function(t){if(t&&(t.city||t.region)){var e=[];return t.city&&e.push(t.city),t.region&&e.push(t.region),t.country&&e.push(t.country),e.join(", ")}}(t)})})}function S(){return U().then(function(t){return Promise.all([t,function(t){if(t&&"number"==typeof t.lng&&"number"==typeof t.lat)return fetch("".concat(k,"?maps=territories&position=").concat(t.lat,",").concat(t.lng)).then(function(t){return t.json()}).then(O).catch(function(t){return console.log(t),[]})}(t)])}).then(function(t){var e=A(t,2),n=e[0];return{territories:e[1],commonName:n.commonName}}).catch(function(t){console.log("Something when wrong",t)})}function R(t,e){var n=document.createElement("a");return n.href=e,n.innerText=t,n.target="__blank",n.className=E.a.link,n}function B(t){if(t&&(t.territories.length||t.commonName)){var e=document.createElement("div");e.className=E.a.container;var n=document.createElement("div");n.className=E.a.header;var r=document.createElement("div");r.className=E.a.headerTitle,r.innerText="Where are we?";var o=document.createElement("button");o.className=E.a.closeButton,o.innerHTML=x.a,o.addEventListener("click",function(){return function(t){t&&0!==t.style.opacity&&(t.style.opacity=0,setTimeout(function(){t&&t.parentElement&&t.parentElement.removeChild(t)},220))}(e)}),n.appendChild(r),n.appendChild(o);var i=document.createElement("div");i.className=E.a.content;var a=document.createElement("p");if(a.className=E.a.text,a.innerText=function(t){var e=function(t){if(0!==t.length)return 1===t.length?"".concat(t[0]," country"):"".concat(t.slice(0,-1).join(", ")," and/or ").concat(t[t.length-1]," country")}(t.territories),n="Your IP indicates that you may be";return!t.commonName&&e?"".concat(n," on ").concat(e,"."):t.commonName&&!e?"".concat(n," in ").concat(t.commonName,"."):"".concat(n," on ").concat(e,", other wise known as ").concat(t.commonName,".")}(t),i.appendChild(a),t.territories.length>=0){var s=document.createElement("p");s.className=E.a.disclaimer,function(t){[document.createTextNode("The information presented here is derived from the maps at "),R("native-land.ca","https://native-land.ca/"),document.createTextNode(" cross-referenced by data from "),R("ipinfo.io","https://ipinfo.io/"),document.createTextNode(". This is not authoritative or representative and should be approached critically. \n      Do you know who the traditional custodians are of the land on which you stand?")].forEach(function(e){return t.appendChild(e)})}(s),i.appendChild(s)}e.appendChild(n),e.appendChild(i),document.body.appendChild(e),setTimeout(function(){e.style.opacity=1},0)}}S().then(B)}]);
```

Comment:
My bad for writing a script which deletes this - but we not want to commit this deletion - as www.frontyardprojects.org is relying on it at the moment.

Reply:
Ah yep, added back in!